### DISCUSSION TITLE
where is config file of input leap?

### DISCUSSION THREAD

**Discussion by littleblack111** [input-leap/input-leap#2019] [Q&A] [answered]
where do i edit file that have content like https://github.com/input-leap/input-leap/blob/master/doc/input-leap.conf.example-advanced

is it ~/.config/InputLeap/InputLeap.conf? It have other stuff that doesnt seem to look alike

**Comment by wfg** [comment, marked as answer]
I have mine at `~/.config/InputLeap/input-leap.conf`. This is the default place for Linux installations. You may put it elsewhere and specify the path with the `--config` flag when running the binary.
